esy-ocaml-old/esy-old
# [WIP] Do NOT rebuild local dependencies if no changes happened since the la…

…st build.



**Summary**
Right now for local dependencies (symlinks or local file:// dependencies) esy build will rebuild them every time even if nothing changed, because we don't detect if anything changed.
See: https://github.com/jordwalke/esy-issues/issues/80

**Test plan**
Still a Work In Progress.
But I'll test the following cases and their variations:
- Build, make changes to local dep, Build again => Rebuild of local dep should happen
- Build, make NO changes to local dep, Build again => Rebuild of local dep should NOT happen
